plugins/legacy-plugin-chart-pivot-table/src/PivotTable.js

```diff
@@ -44,25 +45,37 @@ const propTypes = {
 };
 
 function PivotTable(element, props) {
-  const { data, height, columnFormats, numberFormat, numGroups, verboseMap } = props;
+  const { data, height, columnFormats, dateFormat, numberFormat, numGroups, verboseMap } = props;
 
   const { html, columns } = data;
   const container = element;
   const $container = $(element);
```

Comment:
jQuery!? 😭

Reply:
Unfortunately pivot table still uses `jQuery.dataTables`.

Reply:
Hey, if it works it works! 😄